exercises/concept/paolas-prestigious-pizza/.meta/design.md

```diff
@@ -0,0 +1,57 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach students how to use `elm/parser`, and to state that this is the idiomatic way of parsing in Elm.
+
+## Learning objectives
+
+- Understand parser pipelines
+- Parsers: Parser, run
+- Building blocks: int, keyword, symbol, end
+- Pipelines: succeed, (|=), (|.), andThen, problem
+- Branches: oneOf, map 
+- Loops: sequence
+- Whitespace: spaces
+- Chompers: getChompedString, chompWhile
+
+## Out of scope
+
+- Understand backtracking
+- Understand tracking context
```

Comment:
@ceddlyburge I didn't cover backtracking and tracking context, I estimated that the basics we already complex enough.